### ISSUE TITLE
Close icon is almost invisible on the 'Set schedule' dialog window in dark theme

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#11374]
Initially reported by @albinazs as part of [WCAG 2.2 AAA* audit of the Wagtail admin – Nov 2023](https://github.com/wagtail/wagtail/discussions/11180).

In our dialog modals, _Close icon is almost invisible in dark theme_

This is failure of _1.4.11 Non-text Contrast_:

> Visual objects must have a contrast ratio of 3:1 against adjacent color(s)

Code in question:

```html
<button type="button" class="w-dialog__close-button" aria-label="Close dialog" data-action="w-dialog#hide">
                <svg class="icon icon-cross w-dialog__close-icon" aria-hidden="true"><use href="#icon-cross"></use></svg>
            </button>
```

---

## Solution

Probs an easy fix? Use a different color / correct CSS style override.

**Comment by Taherpatrawala** [comment]
Hello there @thibaudcolas,  can I work on this issue..? :)

**Comment by zerolab** [comment]
Is this related to #11306 ?

**Comment by thibaudcolas** [comment]
Thanks @zerolab, yes :) closing as dupe, I missed that one.
